docs/_articles/ja/einstein/einstein-apex.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Apex Code Generation with Einstein for Developers
+lang: en
+---
+
+## Generate Apex Code
+
+Use the Einstein for Developers side bar to write a question or an instruction that describes the task for which you'd like a receive an Apex code suggestion and press **Ask**. Copy the code suggestion you receive and paste it into an Apex file to use it as a "starter" code.
+
+![Side bar code generation](../../../images/einstein-sidebar.png)
```

Comment:
Let's update this image once we have the latest version of the model deployed - it has `apex` in backticks at the top of the output

Reply:
all your suggestions have been resolved. pending only image updates now. @AnanyaJha